repository/repository.go

```diff
@@ -2,32 +2,33 @@ package repository
 
 import (
 	"database/sql"
-	"github.com/Baraulia/AUTHENTICATION_SERVICE/pkg/logging"
-	"github.com/Baraulia/AUTHORIZATION_SERVICE/model"
+	auth_proto "stlab.itechart-group.com/go/food_delivery/authorization_service/GRPC"
+	_ "stlab.itechart-group.com/go/food_delivery/authorization_service/docs"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/model"
+	"stlab.itechart-group.com/go/food_delivery/authorization_service/pkg/logging"
 )
 
 //go:generate mockgen -source=repository.go -destination=mocks/repository_mock.go
 
-type Authorization interface {
-	GetUser(email, password string) (*model.User, error)
-}
+type RolePerm interface {
+	GetRoleById(id int) (*model.Role, error)
+	GetAllRoles() ([]model.Role, error)
+	CreateRole(role string) (int, error)
+	BindRoleWithPerms(rp *model.BindRoleWithPermission) error
+
+	GetPermsByRoleId(id int) ([]model.Permission, error)
+	CreatePermission(permission string) (int, error)
+	GetAllPerms() ([]model.Permission, error)
 
-type RoleList interface {
-	GetById(id int) (*model.Roles, error)
-	SelectPermission (id int) ([]model.Permission,error)
-	CreateRole(role *model.Role) (*model.Role, error)
-	CreatePermission(permission *model.Permission)(*model.Permission, error)
-	CreateRoleToPermission(rp *model.RoleToPermission)(*model.RoleToPermission, error)
+	BindUserWithRole(user *auth_proto.User) error
```

Comment:
I would use `Add` instead of bind. `AddUserRole` or `AddRoleToUser`. Because you would also need endpoint to remove the role and the word Unbind would sound strange as for me.